doc/iterations/2025.2/lower_coupling_between_project_and_editing_context.adoc

```diff
@@ -0,0 +1,37 @@
+= Lower the coupling between project and editing context 
+
+== Problem
+
+Sirius Web uses the UUID project id as the id of the editing context.
+
+We want to be able to use more complex ids.
+
+== Key Result
+
+The `project id` should be declared as *String* instead of *UUID* in several tables :
+
+- project (PK)
+- semantic_data (FK)
+- representation_meta_data (FK)
+- project_image (FK)
+- nature (FK)
+
+An implementation of the interface `IEditingContextIdProvider` should be implemented in order to return an `editing context id` from a `project_id`.
+
+The default implementation would keep the current behaviour and return the project_id as the new editing context id.
+
+=== Scenario
+
+Everything should work as before the implementation.
+
+
+=== Breadboarding
+
+
+=== Cutting backs
+
+
+== Rabbit holes
+
+
+== No-gos
```

Comment:
Could you add a new section:

=== Acceptance Criteria

- Everything should work as before the implementation
- The editing context id should be the identifier of the semantic data instead of the identifier of the project